### PR TITLE
Bulkrax Improvements

### PR DESCRIPTION
**Pull request by cjcolvar**
Fix issues with admin sets, collections, parsing of multiple valued fields, date fields, and resource types.

### CODE REVIEW

File: app/models/hyku_addons/csv_entry.rb [hunk 1]
```diff
@@ -59,11 +78,31 @@ def add_json_fields
       end
     end
 
+    def add_resource_type
+      resource_type_service = HykuAddons::ResourceTypesService.new(model: parsed_metadata['model'].safe_constantize)
```
Comment (okeen) [inline app/models/hyku_addons/csv_entry.rb:82]: Will this need a safe operator? 

`parsed_metadata['model']&.safe_constantize`

File: app/parsers/hyku_addons/csv_parser.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 module HykuAddons
   class CsvParser < Bulkrax::CsvParser
+    # FIXME: Override to make debugging easier
+    def perform_method
+      return :perform_now if Rails.env == 'development' || Rails.env == 'test'
```
Comment (okeen) [inline app/parsers/hyku_addons/csv_parser.rb:6]: maybe I would replace that if for:

```suggestion
      return :perform_now unless Rails.env.production?
```

WDYT?

**Review by okeen — APPROVED** [review]

File: app/parsers/hyku_addons/csv_parser.rb (hunk 2 above)
Comment (cjcolvar) [reply app/parsers/hyku_addons/csv_parser.rb:6]: Yes, I forgot about that.  I'll get that in the next PR.

File: app/models/hyku_addons/csv_entry.rb (hunk 1 above)
Comment (cjcolvar) [reply app/models/hyku_addons/csv_entry.rb:82]: I think it should always be there but wouldn't hurt to add and might help with debugging.  I'll get it in the next PR.

**Pull request merged**